### ISSUE TITLE
Issues in tests

### ISSUE THREAD

**Issue by ifteda** [kevlee1/cs4500-a1-part2#8]
Getting some issues in tests 4 and 5.
test4(): Methods expecting objects are given primitive types

test5(): length() method is used, but not required by spec. Did you mean to use size()?

**Comment by kevlee1** [comment]
Yeah, I just changed the spec to accommodate for primitive types. I added wrapper classes into array.h and also just changed the tests to reflect the changes

**Comment by frankkulak** [comment]
@kevlee1 Please remove the wrapper classes. Those should not be part of the spec. We can implement those in a helper file if we choose, but including it in your spec makes it so we have to implement them.

**Comment by kevlee1** [comment]
Ok, I will remove them. Do you need any changes for the tests? I changed them to use the wrapper classes.

**Comment by frankkulak** [comment]
@kevlee1 Thanks! Also, there are quite a few issues in the new test file. We cannot use it unless they are fixed, as the file does not compile against your spec. Please only change the test file and not your spec, as this assignment is due in 3 hours. 

Issues:
- lines 82-85: Array does not have a constructor that takes in an int. Please remove these ints.
- line 113: There is no defined `t_false()` method - please use `t_fail()`.
- line 129: `set()` expects a `size_t` followed by an `Object*`, but you are passing  them backwards. Please fix this too.

**Comment by kevlee1** [comment]
@frankkulak Fixed the tests! Let me know if there needs to be any more changes
